app/controller.js

```diff
@@ -1838,6 +1853,19 @@ class MainController {
             }
         } else if (keys === 'alt_space') {
             this.playRegion()
+        } else if (which) {
+            if (which === 219) {
+                this.moveBack()
+            }
+        }
+    }
+
+    moveBack () {
```

Comment:
Why not use wavesurfer.skip?

Reply:
I've changed my code to use .skip, thanks